velox/docs/monitoring/stats.rst

```diff
@@ -18,6 +18,7 @@ analysis.
 
 Memory Arbitration
 ------------------
+
```

Comment:
Let's add a sentence here to explain that these stats are available for all operators.